examples/app_service/app-sample-blueprint.yaml

```diff
@@ -68,18 +48,19 @@ dsl_definitions:
 
   app_config: &app_config
     location: { get_input: location }
-    server_farm_id: sample-plan
+    server_farm_id: { get_attribute:[plan, resource_id] }
     site_config:
       scm_type: LocalGit
       python_version: 3.6
```

Comment:
since we are using get_attribute we should wait for it to finish 
`relationships:
      - type: cloudify.relationships.depends_on
        target: plan`